broadcast_test.go

```diff
@@ -11,25 +14,31 @@ func TestBroadcaster(t *testing.T) {
 	b := NewBroadcaster()
 	for i := 0; i < 10; i++ {
 		sinks = append(sinks, newTestSink(t, nEvents))
-		b.Add(sinks[i])
-		b.Add(sinks[i]) // noop
+		err := b.Add(sinks[i])
+		assert.NoError(t, err)
```

Comment:
Wondering if we (haven't looked closely yet) can do without assertion library). The repo had only one dependency (logrus) which we _probably_ may be able to get rid of; in that case it would have no dependencies at all, which is rather nice

Reply:
Should not be too bad without the library. All the assertions are `NoError` on second review.

Reply:
Yes, if it's indeed only those (and no advanced assertions are needed), that'd be great.

It's _slightly_ more cumbersome to write, but may be worth the effort for library modules like this.  🤗